### PR TITLE
feat: initialize thirdparty snippets

### PR DESCRIPTION
**Pull request by kleyow**
Small PR to initialize the thirdparty snippets. Over time we can add snippets as we develop.

### CODE REVIEW

File: docs/thirdparty-openapi3-snippets.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        enum:
+          - http
+          - https
+        default: https
+components:
+  schemas:
+    AccountAddress:
+      type: string
+      description: |
+        A long-lived unique account identifier provided by the DFSP this MUST NOT
```
Comment (stevenwjy) [inline docs/thirdparty-openapi3-snippets.yaml:24]:
```suggestion
        A long-lived unique account identifier provided by the DFSP. This MUST NOT
```

File: thirdparty/openapi3/schemas/AccountAddress.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+type: string
+description: |
+  A long-lived unique account identifier provided by the DFSP this MUST NOT
```
Comment (stevenwjy) [inline thirdparty/openapi3/schemas/AccountAddress.yaml:3]:
```suggestion
  A long-lived unique account identifier provided by the DFSP. This MUST NOT
```

File: package.json [hunk 3]
```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "buildv1.0": "swagger-cli bundle -t yaml -o docs/fspiop-rest-v1.0-openapi3-snippets.yaml v1.0/openapi3/index.yaml && swagger-cli validate docs/fspiop-rest-v1.0-openapi3-snippets.yaml",
     "buildv1.1": "swagger-cli bundle -t yaml -o docs/fspiop-rest-v1.1-openapi3-snippets.yaml v1.1/openapi3/index.yaml && swagger-cli validate docs/fspiop-rest-v1.1-openapi3-snippets.yaml",
+    "buildThirdparty": "swagger-cli bundle -t yaml -o docs/thirdparty-openapi3-snippets.yaml thirdparty/openapi3/index.yaml && swagger-cli validate docs/thirdparty-openapi3-snippets.yaml",
```
Comment (stevenwjy) [inline package.json:8]: would it be better to name the script something like `build:thirdparty`, `build:v1.0`, and `build:v1.1` ?

Comment (akapur99) [reply package.json:8]: i like the former `build:thirdparty` 
that seems in line re: script format?

Comment (kleyow) [reply package.json]: yep sounds good. changed.

File: thirdparty/openapi3/schemas/AccountAddress.yaml (hunk 2 above)
Comment (kleyow) [reply thirdparty/openapi3/schemas/AccountAddress.yaml]: done.

File: docs/thirdparty-openapi3-snippets.yaml (hunk 1 above)
Comment (kleyow) [reply docs/thirdparty-openapi3-snippets.yaml]: done.

**Pull request merged**